draft-ietf-rats-architecture.md

```diff
@@ -424,6 +424,17 @@ requires another component to feed various values to the TPM.
 Thus, an Attesting Environment in such a case would be the combination
 of the TPM together with whatever component is feeding it the measurements.
 
+## Environment Interactions
+
+As explained in the sections below, the Attesting Environment may be arbitrarily complex.
+It may be distributed in various ways, or may depend upon other environments that collect claims on the environment itself as part of the process.
```

Comment:
What does "distributed" mean here?   Distributing code for an attester?   Maybe pick another term?